### DISCUSSION TITLE
Saving zoomed view after a restart

### DISCUSSION THREAD

**Discussion by jakemkc** [vslinko/obsidian-zoom#59] [Ideas]
Hi all,

I am a heavy Obsidian Outliner and Zoom plugins user. They work nice together and greatly increased my productivity.

Sometimes I have a longer document and I will zoom in to the paragraph to stay focus on it while I am multi-tasking.

However, when I quit and restart Obsidian, I am back to the original page but not the zoomed view.

It's okay to click in again, however, I wonder is it possible to make it remember the zoomed view and save a few clicks?

Best,
Jake

**Comment by tonyxiao** [comment]
This also happens when you navigate away. Super annoying!
